src/config/index.js

```diff
@@ -0,0 +1,11 @@
+module.exports = Object.freeze({
+  NODE_PORT: process.env.NODE_PORT || 3000,
+  MONGO_DB: {
+    HOST: process.env.MONGO_HOST || 'localhost',
+    PORT: process.env.MONGO_PORT || 27017,
+    DATABASE: process.env.MONGO_DB || 'socialShare',
+    get URI() {
+      return `mongodb://${this.HOST}:${this.PORT}/${this.DATABASE}`;
```

Comment:
It looks really better that way to the one i used to do. I get the logic behind, but i don't know how it will work with mlab url for the database. By now i think we can leave it like.

Reply:
We can check how it works and adapt this to work with it. At the end of the day it should use a db url to connect to the db, and in this case it's simply replacing this with the url.